libs/GraphicECS/SFML/Resources/SoundResource.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** SoundResources
+*/
+
+#ifndef SOUNDRESOURCES_HPP_
+#define SOUNDRESOURCES_HPP_
+
+#include <SFML/Audio.hpp>
+#include <filesystem>
+#include "Resource/Resource.hpp"
+#include <unordered_map>
+
+namespace ecs
+{
+    /// @brief This resource class stores a map of Sounds SFML.
+    class SoundResource : public Resource {
+      public:
+        /// @brief Enumeration of all available Sounds
+        enum soundName_e { UNDEFINED, SHOOT, DEATH, EXPLOSION };
+
+        /// @brief Name of map which contains Sounds.
+        using SoundsList = std::unordered_map<soundName_e, std::shared_ptr<sf::Sound>>;
+
+        /// @brief Add a Sound from it's Sound Path passed as parameter
+        /// @param sound_e Enum of the Sound
+        /// @param soundPath The sound path to be used.
+        inline SoundResource(const soundName_e sound_e, const std::filesystem::path &soundPath)
```

Comment:
Can we directly passed a list of sound ? And why not ?

Reply:
Nop the buffer in sfml cannot support a multiple load